test/orange/surf/detail/InvolutePoint.test.cc

```diff
@@ -17,7 +17,9 @@ namespace detail
 {
 namespace test
 {
-using constants::pi;
+constexpr real_type pi{constants::pi};
```

Comment:
This might have been missed as part of the commit "Use 'typical' usage of constant in unit tests "